demos/custom-private-dns/azuredeploy.json

```diff
@@ -28,49 +28,54 @@
         "description": "The DNS name for the public IP address used by the Load Balancer infront of the DNS servers"
       }
     },
-    "assetLocation": {
+    "adTemplateLocation": {
       "type": "string",
       "metadata": {
-        "description": "The location of resources, such as templates and scripts, that this script depends on"
+        "description": "The location of template to create HA Domain Controllers"
       },
-      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/custom-private-dns"
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/application-workloads/active-directory/active-directory-new-domain-ha-2-dc/"
```

Comment:
change this to: 

https://github.com/Azure/azure-quickstart-templates/tree/master/modules/active-directory-new-domain/0.9

I'm not 100% certain it will work, but seems like it should - and thats the location for shareable modules